server/odc-migrate/src/main/resources/migrate/common/V_4_3_1_4__add_logical_database.sql

```diff
@@ -0,0 +1,57 @@
+alter table connect_database modify column connection_id bigint(20) DEFAULT NULL COMMENT 'refernce to connect_connection.id';
```

Comment:
seems dupliated 3 copy of modify column connection_id?

Reply:
My bad I removed the duplicated sqls.